src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -861,7 +861,7 @@ private async Task<FormulaValue> PostProcessResultAsync(FormulaValue result, Bas
         {
             ExpressionError er = null;
 
-            if (result is ErrorValue ev && (er = ev.Errors.FirstOrDefault(e => e.Kind == ErrorKind.Network || e.Kind == ErrorKind.InvalidJSON)) != null)
+            if (result is ErrorValue ev && (er = ev.Errors.FirstOrDefault(e => e.Kind == ErrorKind.Network || e.Kind == ErrorKind.InvalidJSON || e.Kind == ErrorKind.InvalidArgument)) != null)
```

Comment:
>|| e.Kind == ErrorKind.InvalidArgument [](http://example.com/codeflow?start=142&length=38)

Can you check with Luc, who wrote this code, if this is ok?

Reply:
That should be ok

Reply:
Luc has confirmed that this should be good.